### PR TITLE
fix: setting .env variables on subservice

### PR DESCRIPTION
**Pull request by kevinacuna**

### CODE REVIEW

File: doc/manuals/simulation-configuration-file.md [hunk 1]
```diff
@@ -523,6 +523,22 @@ Last but not least, the [`./examples`](https://github.com/telefonicaid/fiware-de
...
+``` tree
+├── electricvehicle         # Subservice directory (simulation has to have the same configuration)
+│   ├── .env.json           # Environment variables (just for this directory and all the inner simulation)
+│   ├── simulation1.json    # simulation1.json tests
+│   └── simulation2.json    # simulation2.json tests
+│   └── ...
+├── bathwaters              # Subservice directory (simulation has to have the same configuration)
+│   ├── .env.json           # Environment variables (just for this directory and all the inner simulation)
+│   ├── simulation1.json    # simulation1.json tests
+│   └── simulation2.json    # simulation2.json tests
```
Comment (fgalan) [inline doc/manuals/simulation-configuration-file.md:537]:
```suggestion
│   ├── .env                # Environment variables (just for this directory and all the inner simulation)
│   ├── simulation1.json    # simulation1.json tests
│   └── simulation2.json    # simulation2.json tests
│   └── ...
├── bathwaters              # Subservice directory (simulation has to have the same configuration)
│   ├── .env                # Environment variables (just for this directory and all the inner simulation)
│   ├── simulation1.json    # simulation1.json tests
│   └── simulation2.json    # simulation2.json tests
```

Unsure, but I'd say that's the right name

File: doc/manuals/simulation-configuration-file.md [hunk 2]
```diff
@@ -523,6 +523,22 @@ Last but not least, the [`./examples`](https://github.com/telefonicaid/fiware-de
 The simulation configuration file may use an `.env` file to set configuration in the form of environment variables. This can be used to avoid setting
 sensitive information, and be executed in whatever environment, having different configurations and not been needed to change anything on code.
 
+It is important to know that, to use environmet variables is neccessary to create an .env file with all the environment variables that the simulation file will use at the same directory root, having a result as next:
```
Comment (fgalan) [inline doc/manuals/simulation-configuration-file.md:526]: Maybe simpler this way:

```suggestion
Note that the `.env` file used in a given simulation file is the one that is located in the same directory where the simulation file is. For instance:
```

File: bin/fiwareDeviceSimulatorCLI [hunk 3]
```diff
@@ -37,7 +37,20 @@ var path = require('path');
 var request = require('request');
```
Comment (fgalan) [inline bin/fiwareDeviceSimulatorCLI:37]: New entry should be included in CNR file. Maybe something like this:

```
- Fix: per-directory .env files
```

Comment (kevinacuna) [reply bin/fiwareDeviceSimulatorCLI:37]: Added

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**